src/multi.c

```diff
@@ -230,6 +230,7 @@ void execCommand(client *c) {
         /* Commands may alter argc/argv, restore mstate. */
         c->mstate.commands[j].argc = c->argc;
         c->mstate.commands[j].argv = c->argv;
+        c->mstate.commands[j].argv_len = c->argv_len;
```

Comment:
not sure if this needs to be changed
```suggestion
        /* Commands may alter argc/argv/argv_len, restore mstate. */
        c->mstate.commands[j].argc = c->argc;
        c->mstate.commands[j].argv = c->argv;
        c->mstate.commands[j].argv_len = c->argv_len;
```

Reply:
i don't think the comment needs an update